tableauserverclient/models/job_item.py

```diff
@@ -5,14 +5,16 @@
 
 
 class JobItem(object):
-    def __init__(self, id_, job_type, progress, created_at, started_at=None, completed_at=None, finish_code=0):
+    def __init__(self, id_, job_type, progress, created_at, started_at=None,
+                 completed_at=None, finish_code=0, notes=[]):
```

Comment:
Set `notes` equal to `None` in the `__init__` and then if you want to to be an empty list, you can do that in the assignment to `self._notes` with `if notes is None` but I think you can just leave this as `None` generally. Up to you!

Reply:
I set the default as an empty list so that it would always be iterable without having to check for a `NoneType` first. Ultimately I see that difference as a matter of preference. I can make a change to initialize it as `None` and return an empty iterable if it is `None`, though. That seems like an extra step to the same end. Is the preference that it not always be iterable?

Reply:
Following advice from here https://nikos7am.com/posts/mutable-default-arguments/ <- I think it might not be an issue in this particular case but it is the recommend Pythonic way to do it, and the style I generally follow.

As for iterable by default, makes sense to me for making it a `[]` after object initialization